.github/workflows/create_release.yml

```diff
@@ -0,0 +1,144 @@
+name: "Create release"
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
Are you sure this is the right trigger? 
This will publish a new version whenever any PR is closed.
I think it should be this to release on every push to main - provided we want this
```
on:
  push:
    branches:
      - main
```